src/runtime_src/core/edge/user/graph_object.cpp

```diff
@@ -1,168 +1,305 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#ifdef XRT_ENABLE_AIE
```

Comment:
This whole file is under XRT_ENABLE_AIE flag. Can you please update CMAKe to reflect this instead of whole file under ifdef.